### ISSUE TITLE
Reset the Game

### ISSUE THREAD

**Issue by rmccaskill** [FrederictonMakerspace/LaserMaze#3]
```gherkin

  Scenario: Resetting the game
    Given A game isn't in progress
    When 
         The reset button is pushed
    Then 
         Reset the game timer on the UI
         Play a cool reset/power up sound
         Turn the lasers back on
         Enable the 'Start' button for the player to press
```
---

**Comment by rmccaskill** [comment]
Need to push a 'reset' message to the Arduino so that it knows that it's time to 'reset' in case there is state being kept there.

**Comment by TheaterMan72** [comment]
push a 2 to the arduino and I will handle it to reset the lasers and sensors

**Comment by rmccaskill** [comment]
e4b2f0356dcd22635f26d19009bb2f0a72e34467 now provides a 'reset' button which resets the UI and plays a 'power up' sound but doesn't send anything to the arduino (yet)

**Comment by rmccaskill** [comment]
Closed via commit 00df0d3344ddd393bb9c0310a7a6db38df86923f
